lib/ontologies_linked_data/serializers/json.rb

```diff
@@ -6,6 +6,21 @@ class JSON
       CONTEXTS = {}
 
       def self.serialize(obj, options = {})
+        
+        result_lang = ''
+
+        if obj.submission
+          
+          obj.submission.bring :naturalLanguage
+          langauges = get_submission_languages(obj.submission.naturalLanguage)
+                  
+          # intersection of the two arrays , if the requested language is not :all
+          result_lang = options[:lang] == :all ? langauges : options[:lang] & langauges
+          result_lang = result_lang.first if result_lang.length == 1
```

Comment:
result_lang is not used after?

Reply:
line `58` =>  `hash['@context']['@language'] = result_lang`